app/containers/Login/saga.js

```diff
@@ -142,6 +149,31 @@ export function* loginWithScatterWorker() {
   }
 }
 
+export function* sendReferralCode(
+  accountName,
+  referralCode,
+  eosService,
+  error,
+) {
+  const isUserIn = yield call(isUserInSystem, referralCode, eosService);
+
+  if (isUserIn) {
+    try {
+      yield call(inviteUser, accountName, referralCode, eosService);
+    } catch (err) {
+      return;
```

Comment:
Errors should always be logged.

In this case, we should not complete registration if could not register invited user.